AdobeStockImageAdminUi/Test/Mftf/Data/AdobeStockConfigData.xml

```diff
@@ -15,18 +15,4 @@
         <data key="path">cms/wysiwyg/enabled</data>
         <data key="value">enabled</data>
     </entity>
-    <entity name="AdobeStockConfigDataApiKey">
-        <data key="path">adobe_stock/integration/api_key</data>
-        <data key="value">{{_ENV.ADOBE_STOCK_API_KEY}}</data>
-    </entity>
-    <entity name="AdobeStockConfigDataPrivateKey">
```

Comment:
I'd also use the Data entity to store the `_CREDS.magento/ADOBE_STOCK_USER_EMAIL` and `_CREDS.magento/ADOBE_STOCK_USER_PASSWORD`, instead of directly using them into Action Groups.

Reply:
My only concern is that, according to the [MFTF documentation, using Credentials only works with three MFTF elements](https://devdocs.magento.com/mftf/docs/credentials.html#use-credentials-in-a-test): `fillField`, `magentoCLI`, and `createData`. But, I can try to use them with these other elements and see if things still work.

Reply:
I have confirmed using the entity/data key value pairs with credentials works, so I will revert this part of the PR.

Reply:
Actually let's first check what the MFTF experts say about using credentials in entity/data elements https://github.com/magento/magento2-functional-testing-framework/issues/506

Reply:
OK, I have reverted back to using data and entity elements with secure credentials. If the MFTF team gets back to use with a definitive answer that we should not do this, we can update the code later.